go/store/prolly/tree/node_cursor.go

```diff
@@ -324,61 +333,75 @@ func (cur *Cursor) search(item Item, cb CompareFn) (idx int) {
 	return idx
 }
 
-// todo(andy): improve the combined interface of Advance() and advanceInBounds().
-//  currently the returned boolean indicates if the cursor was able to Advance,
-//  which isn't usually useful information
+// jumpToEnd sets the cursor's index to the node count.
+func (cur *Cursor) jumpToEnd() {
```

Comment:
maybe something like `invalidate()` to distinguish this method from `skipToNodeStart/End()`